### ISSUE TITLE
M3u8::Playlist.read("path/to/file") fails silently

### ISSUE THREAD

**Issue by jbodah** [sethdeckard/m3u8#20]
Had a few minutes of head scratching when I tried calling `M3u8::Playlist.read` with a path before I realized I needed an IO object. Calling `read` with a path doesn't throw an error and returns a playlist regardless

```
irb(main):002:0> M3u8::Playlist.read("~/Desktop/long_hls")
=> #<M3u8::Playlist:0x007fb77388de20 @version=nil, @sequence=0, @cache=nil, @target=10, @type=nil, @iframes_only=false, @independent_segments=false, @items=[]>
```

**Comment by sethdeckard** [comment]
Good catch, I agree that the reader should raise an error if the first line does not contain the #EXTM3U tag.

**Comment by sethdeckard** [comment]
Fixed in v0.8.0
